nixos/modules/services/networking/nextdns.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nextdns;
+in {
+  options = {
+    services.nextdns = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether or not to enable the NextDNS DNS/53 to DoH Proxy service.";
+      };
+      arguments = mkOption {
+        type = types.separatedString " ";
+        example = "-config 10.0.3.0/24=abcdef";
```

Comment:
I suggest making this a `types.listOf types.str` instead, then you can use `lib.escapeShellArgs` in the `ExecStart` below not have to worry about shell escaping

Reply:
Hey that's pretty cool - thanks!